packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -52,6 +56,18 @@ export const getDocs: () => JSX.Element = () => {
       </Description>
       <Source code={cssSnippet} />
 
+      <Heading>Running in a Mobile browser</Heading>
+      <Description>
+        CallComposite has been designed for both desktop views and mobile views. By default the CallComposite UI is
+        optimized for desktop views and does not do any automatic detection if it is running on a mobile. To have an
```

Comment:
Slightly confusing description. It is not clear if this flag changes the layout to a mobile layout immediately or starts an automatic detection and changes if it detects a mobile.

Reply:
Thanks will update (it changes immediately, we do no detection ever)

Reply:
I love that we don't handle the detection part.
One less thing to worry about.